src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+function wrapHeadlines() {
+  const treeItems = document.querySelectorAll('.tree li');
+
+  treeItems.forEach((li) => {
+    const textNode = Array.from(li.childNodes).find(
+      (node) => node.nodeType === Node.TEXT_NODE,
+    );
+
+    if (textNode) {
+      const span = document.createElement('span');
+
+      span.textContent = textNode.textContent.trim();
+      li.replaceChild(span, textNode);
+    }
+  });
+}
+
+function toggleTree(even) {
```

Comment:
The parameter name `even` in the `toggleTree` function seems to be a typo. It should likely be `event` to accurately represent the event object being passed in.